src/utils/utils.test.ts

```diff
@@ -0,0 +1,94 @@
+import { CurrentQuestion } from '../Components/Quiz/CurrentQuestion';
+import { quizdatabase } from '../database/database';
+import {
+	getAttemptedPercentage,
+	getRightAnswers,
+	getTotalScore,
+	getWrongAnswers,
+	getCurrentQuestion,
+	styleRightAndWrongAnswers,
+	isOptionSelected,
+	isRightAnswer,
+} from './utils';
+
+const result = {
+	quizId: '123',
+	resultArray: [
+		{
+			id: '1',
+			hasTaken: true,
+			selectedOption: '999',
+			correctOption: '999',
+		},
+		{
+			id: '2',
+			hasTaken: false,
+			selectedOption: '111',
+			correctOption: '112',
+		},
+	],
+};
+
+const currentQuiz = quizdatabase[0];
+
+describe('testing utility functions of scoreboard', () => {
+	test('should return the count of questions that has been attempted by the user', () => {
+		const attemptedQuestionsCount = getAttemptedPercentage(result.resultArray);
+		expect(attemptedQuestionsCount).toEqual(1);
```

Comment:
You could have used `toBe` here. Since it's a primitive type. This approach also looks to be fine.